collections/ansible_collections/massopencloud/esi/roles/network/meta/argument_specs.yaml

```diff
@@ -0,0 +1,12 @@
+argument_specs:
+  main:
+    options:
+      network_state:
+        type: "str"
+        required: true
+        choices:
+        - present
+        - absent
+      network_name:
+        type: "str"
+        required: true
```

Comment:
`network_identifier` should be declared here, and I think `network_name` shouldn't be required because it's not used in case of deletion.

Reply:
ah, good catch - thanks! should be fixed now